peekingduck/pipeline/nodes/output/media_writer.py

```diff
@@ -83,15 +86,37 @@ def _prepare_writer(self, filename: str, img: np.array, fps: int) -> None:
         self._file_name = filename  # type: ignore
         self._file_path = os.path.join(  # type: ignore
             self._output_dir, filename)  # type: ignore
+        self.file_path_with_timestamp = self._append_datetime_filename(filename) #type: ignore
```

Comment:
What are your thoughts? Should `self.file_path` and `self._file_name` be deprecated with the introduction of `datetime_filename`? Could it be good to migrate `file_path`  to `datetime_filename`?

Reply:
I deprecated self.file_path and shifted self.filename into _append_datetime_filename. Since self.file_path will not be used anymore,

Reply:
Thanks for the changes.